### ISSUE TITLE
`bk configure --keyring-keychain=login` doesn't store tokens in macos Keychain

### ISSUE THREAD

**Issue by filipVisko** [buildkite/cli#87]
When using `bk configure --keyring-keychain=login`, bk will attempt to save your buildkite api token using a home directory keyring instead of the intended macos Keychain.

```
> bk --version
1.1.0
> bk configure --keyring-keychain=login                                               
──── Ok! Let's get started with configuring bk 🚀
──── Configuring Buildkite GraphQL credentials
Create a GraphQL token at https://buildkite.com/user/api-access-tokens/new. Make sure to tick the GraphQL scope at the bottom.

GraphQL Token: redacted@redacted.com ✅

Enter passphrase to unlock /Users/redacted/.buildkite/keyring/:
...
```

**Comment by thundergolfer** [comment]
This has confused me a bit, too. I'd expect the the credentials would be configured so that I can add the keyphrase to my OSX keychain and have the auth become transparent to me. 

Right now I'm having to type in my keyphrase every time I invoke a `create` command. Setting an empty passphrase doesn't appear to work.

**Comment by snasirca** [comment]
We're experiencing this same issue. We keep having to enter our password when trying to use `bk`.

**Comment by dontlaugh** [comment]
Asking for a passphrase every time makes this cli unusuable, unfortunately.

And keyring apps are secure, but too much of pain. Plus they're not always available in every environment. 

A nicer approach is the session tokens approach. Like the `aws` STS api. Also look into how the Bitwarden cli works. It has a nice approach.

Of course, generating these tokens means you all will need to run an api or use a service for that.

**Comment by pda** [comment]
I suspect this is “solved” by removing keyring in #115 /  https://github.com/buildkite/cli/releases/tag/v2.0.0 😅
